Doc/library/multiprocessing.rst

```diff
@@ -1049,27 +1049,25 @@ Miscellaneous
 
 .. function:: get_context(method=None)
 
-   Return a context object which has the same attributes as the
-   :mod:`multiprocessing` module.
+   Return a context object with the same attributes as the :mod:`multiprocessing` module.
```

Comment:
We tend to avoid unrelated changes, pleas revert the change to the summary line.